report/styled_par.go

```diff
@@ -0,0 +1,69 @@
+/*
+ * This example showcases the creation of styled paragraph
+ * The output is saved as styled_paragraph.pdf which illustrates some of the features
+ * of the creator.
+ */
+/*
+ * NOTE: This example depends on github.com/unidoc/unipdf/v3/creator, MIT licensed,
+ *       and github.com/unidoc/unipdf/v3/model,
+ *       Apache-2 licensed.
+ */
+
+package main
+
+import (
+	"log"
+
+	"github.com/unidoc/unipdf/v3/creator"
+	"github.com/unidoc/unipdf/v3/model"
+)
+
+func main() {
+
+	c := creator.New()
+	c.NewPage()
+
+	fontRegular, _ := model.NewStandard14Font(model.HelveticaName)
+	fontBold, _ := model.NewStandard14Font(model.HelveticaBoldName)
+
+	//Styled paragraphs ***********************************************
+	//Setting heading style
+	stChap := c.NewChapter("Styled Paragraphs")
+	stChap.GetHeading().SetMargins(0, 0, 20, 0)
+	stChap.GetHeading().SetFont(fontBold)
+	stChap.GetHeading().SetFontSize(18)
+	stChap.GetHeading().SetColor(creator.ColorRed)
+
+	//Creatign new styled paragraph
+	stylPar := c.NewStyledParagraph()
+	stylPar.SetLineHeight(3) //setting line height
+
+	boldStyle := c.NewTextStyle() //creating new style
+
+	boldStyle.Font = fontBold                                                                                       //setting chunk's font style
+	boldStyle.Color = creator.ColorGreen                                                                            //setting chunk's color
+	chunk := stylPar.Append("This text is bolded and is in green color. We're showing how styled paragraphs work.") //setting chunk's text
+	chunk.Style = boldStyle                                                                                         //setting chunk's style
+
+	normStyle := c.NewTextStyle()                                                                                                                                               //creating new style
```

Comment:
We usually have comments above the code.  Please check the other examples, style guide and unipdf codebase and try to keep consistent.
No need to add comments that are obvious.